hvplot/converter.py

```diff
@@ -703,6 +704,15 @@ def _process_data(self, kind, data, x, y, by, groupby, row, col,
 
             if groupby:
                 groupby = [g for g in groupby if g not in grid]
+
+            if not groupby and not grid:
+                # Set a title for coords of size 1 (e.g. da.isel(lon=0).hvplot())
+                if isinstance(da, xr.DataArray):
+                    self._title = da._title_for_slice()
+                elif isinstance(da, xr.Dataset):
+                    one_var = list(da.data_vars)[0]
+                    self._title = da[one_var]._title_for_slice()
```

Comment:
The DataArray title seems clear, but I'm less sure about the Dataset one:  
- It seems to be assuming that there is always at least one data_var; is that always true? I.e., it's not possible to have a Dataset with coordinates but no data values? If so, fine; if not, should first check that the list is nonempty.
- Why does it only use the title from the first data_var? Is it known that will be the only one plotted?  Or should it concatenate all of the titles? 

If it's clear that there is always exactly one data_var, then these considerations don't apply but there should be a comment explaining why that's a safe assumption.

Reply:
> It seems to be assuming that there is always at least one data_var; is that always true? I.e., it's not possible to have a Dataset with coordinates but no data values? If so, fine; if not, should first check that the list is nonempty.

I checked and it's actually possible to have a Dataset with no data values. The code in that function already assumed (line 660) that Dataset objects have at least one data_var. I'll add a check which will raise an error in the event of a Dataset with no data_var.

> Why does it only use the title from the first data_var? Is it known that will be the only one plotted? Or should it concatenate all of the titles?

That was a simplification, I've found a way to plot the scalar coords of a Dataset directly.